### ISSUE TITLE
flow stopped due to missing node types

### ISSUE THREAD

**Issue by Bettelmage** [alex6o/node-red-contrib-dynamic-dimmer#4]
Hey!

First of all thanks for creating this node!

The current version I have installed is 1.1.0 and I installed it via the palette.
My flow is not deploying correctly with error "flow stopped due to missing node types: dynamic-dimmer". 

Any clue what is wrong here? Bug or problem on my side?

Thanks in advance

**Comment by alex6o** [comment]
Hi! I can reproduce this error when I try to use the example flow from `/samples/basics/dynamic_dimmer_basics.json ` without installing the node. Could you please check if the node shows up under the installed nodes? 

Which node red version do you use?

**Comment by Bettelmage** [comment]
I'm running version 1.0.4

![DynamicDimmerError](https://user-images.githubusercontent.com/6348432/85071327-9da21c80-b1b7-11ea-9176-dd784aaa8a45.png)

That's the way it looks in the palette now.

**Comment by alex6o** [comment]
hi! I retested the installation with the included docker-compose and 1.0.4, but it seems to work as expected. could you please have a look on the logs of your node red instance when reinstalling the node? You should get something similar to the following lines:
```
node-red_1  | 21 Jun 06:42:37 - [info] Installing module: node-red-contrib-dynamic-dimmer, version: 1.1.0
node-red_1  | 21 Jun 06:42:57 - [info] Installed module: node-red-contrib-dynamic-dimmer
node-red_1  | 21 Jun 06:42:58 - [info] Added node types:
node-red_1  | 21 Jun 06:42:58 - [info]  - node-red-contrib-dynamic-dimmer:dynamic-dimmer
node-red_1  | 21 Jun 06:42:37 - [info] Installing module: node-red-contrib-dynamic-dimmer, version: 1.1.0
node-red_1  | 21 Jun 06:42:57 - [info] Installed module: node-red-contrib-dynamic-dimmer
node-red_1  | 21 Jun 06:42:58 - [info] Added node types:
node-red_1  | 21 Jun 06:42:58 - [info]  - node-red-contrib-dynamic-dimmer:dynamic-dimmer
```

**Comment by alex6o** [comment]
I will close the issue due to inactivity. Feel free to reopen the issue if needed!
